docs/resources/service_compute.md

```diff
@@ -556,3 +552,562 @@ Fastly Service can be imported using their service ID, e.g.
 ```
 $ terraform import fastly_service_compute.demo xxxxxxxxxxxxxxxxxxxx
 ```
+
+<!-- schema generated by tfplugindocs -->
+## Schema
```

Comment:
@phamann this is actually an example of where the generated Schema works quite well and so we might not want to keep our own curated "Arguments Reference".

Where things get complicated is when the tool fails to pull in the `Description` field values (which happens in a few files, and the behaviour is inconsistent as noted in [this issue I reported](https://github.com/hashicorp/terraform-plugin-docs/issues/28)).

Reply:
Also, it gets awkward if there is ever a scenario where we keep the auto-generated Schema section but we still want to add in some additional 'contextual' information.